lib/index.d.ts

```diff
@@ -0,0 +1,104 @@
+// Type definitions for panel 0.10.0
+// Project: panel
+// Definitions by: Mixpanel (https://mixpanel.com)
+import {VNode} from 'snabbdom';
+import {WebComponent} from 'webcomponent';
+
+export class Component extends WebComponent {
+    /**
+     *  Defines the state of the component, including all the properties required for rendering.
+     */
+    state: any;
+    /**
+     *  Defines standard component configuration.
+     */    
+    config: Component.ComponentConfigOptions;
+    /**
+     *  Template helper functions defined in config object, and exposed to template code as $helpers.
+     *  This getter uses the component's internal config cache.
+     */
+    helpers: any;
+    /**
+     * Called when an instance of the Component is attached to the DOM.
+     */
+     attachedCallback(): void;
```

Comment:
so this is all changing in [the upgrade to Custom Elements v1](https://github.com/mixpanel/panel/pull/18). I'm not sure how the ts conventions generally work, but all the element lifecycle methods (`...Callback()`) are inherited/overridden from HTMLElement; do they need explicit documentation in this class?

Also your indentation seems to have changed starting here.